rotkehlchen/api/v1/schemas.py

```diff
@@ -177,6 +177,12 @@
 log = RotkehlchenLogsAdapter(logger)
 
 
+class AssetValueThresholdSchema(Schema):
+    usd_value_threshold = AmountField(
+        load_default=None,
+    )
```

Comment:
We have those endpoints documented in `docs/api.rst`. Can you also update the documentation there? I forgot to tell you this in the previous review. Since you added a new field it should be documented

```suggestion
class AssetValueThresholdSchema(Schema):
    usd_value_threshold = AmountField(load_default=None)
```